Description: Use HTML 4.0 by default
 This patch sets HTML 4.0 by default.
Author: Adrian Bunk <bunk@fs.tum.de>
Bug-Debian: http://bugs.debian.org/223436

--- latex2html-2008-debian1.orig/l2hconf.pin
+++ latex2html-2008-debian1/l2hconf.pin
@@ -590,7 +589,7 @@ $DEBUG = 0;
 # -html_version
 # The default HTML version to be produced
 #
-$HTML_VERSION = '3.2';
+$HTML_VERSION = '4.0';
 
 
 # -no_math

